rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -196,54 +209,113 @@ class NodeInfoServiceImpl(
       val machineInventories = MutMap[String, LDAPEntry]() // machine_dn -> entry
 
       val n1 = System.currentTimeMillis
```

Comment:
n1 is actually a very bad name (I'm used to "tN"). I don't like too much long variable name for short leaving variable, they tend to clutter the code and make it less readable (it's the same for function like map/filter on collection)